docs/getting-started-9c-with-the-unity-editor/getting-started-9c-with-the-unity-editor.md

```diff
@@ -0,0 +1,256 @@
+
+### 2. Run the Fork and clone the Nine Chronicles repository.
+
+- Click `File > Clone` menu.
+
+  <img
+    alt="Fork: `File > Clone` menu"
+    src="./images/fork-menu-file-clone.png"
+    style="width:50vw" />
+
+- Enter `URL` and `Location` ant click the `Clone` button.
+  - URL: `https://github.com/planetarium/NineChronicles.git`
+  - Location: In the image below, I spesified it as "`/Users/seungmin/Repositories/NineChronicles`".
+
+  <img
+    alt="Fork: `Clone` popup"
+    src="./images/fork-clone-popup.png"
+    style="width:50vw" />
+
+### 3. Checkout to the main branch.
+
+- Right-click the `Remotes > Origin > main` branch and click the `Checkout 'origin/main'...` menu.
+
+  <img
+    alt="Fork: `Remotes > Origin > main` -> `Checkout 'origin/main'...` menu"
+    src="./images/fork-checkout-main.png"
+    style="width:50vw" />
+
+- In the following popup, click the `Track` button.
+
+  <img
+    alt="Fork: `Track` popup"
+    src="./images/fork-track-popup.png"
+    style="width:50vw" />
+
+# Play the Nine Chronicles with the Unity editor
+
+## Open the Nine Chronicles project with the Unity Hub
+
+### 1. Run the Unity Hub.
+
+Unity account creation and licensing are not covered here. You can also proceed for free(Unity Personal License).
+
+### 2. Open the Nine Chronicles project.
+
+- Click the `Projects > Open` button when running for the first time.
+
+  <img
+    alt="Unity Hub: `Projects > Open` button"
+    src="./images/unity-hub-projects-open.png"
+    style="width:50vw" />
+
+- Select the `/nekoyume` folder in the location where you cloned the Nine Chronicles repository and click the `Open` button.
+
+  <img
+    alt="Unity Hub: `Open Project` popup"
+    src="./images/unity-hub-projects-open-popup.png"
+    style="width:50vw" />
+
+## Configure `clo.json`
+
+The clo.json file is a file that sets various options related to the blockchain network.
+
+### 1. Create a clo.json file in `/nekoyume/Assets/StreamingAssets/` based on the cloned path.
+
+### 2. Fill in the clo.json file as shown below and save it.
+
+```json
+{
+    "GenesisBlockPath": "https://download.nine-chronicles.com/genesis-block-Nine Chronicles-main",
+    "NoMiner": true,
+    "RpcClient": true,
+    "RpcServerHost": "9c-main-rpc-1.nine-chronicles.com",
+    "RpcServerPort": 31238,
+    "ApiServerHost": "https://api.9c.gg/graphql"
+}
+```
+
+## Prepare the key in the `Key Store` path
+
+In order to play the Nine Chronicles, you need a Private Key to participate in the blockchain network.
+If you've already played the Nine Chronicles, that means you already have a Private Key and a Protected Private Key.
+
+- Private Key: A key required to sign a transaction.
+- Protected Private Key: This is the encrypted file of the private key.
+
+### 1. Generate Private Key
+
+- Skip this step and go to the [Play](#play) step to use the step to generate a private key.
```

Comment:
I think this sentence means someone who already has private key does't need to create private key but just use existing one.
If it's right, you should move this sentence to heading 2 section and show how to locate `key store path`.
It's awkward to notice of skipping whole process inside substep.

Reply:
@U-lis OK I'll rewrite it.